xml/schema/Core/OptionGroup.xml

```diff
@@ -60,16 +73,20 @@
   </field>
   <field>
     <name>is_reserved</name>
-    <title>Option Group Is Reserved?</title>
+    <title>Option Group Is Reserved</title>
     <type>boolean</type>
     <default>1</default>
     <required>true</required>
     <comment>Is this a predefined system option group (i.e. it can not be deleted)?</comment>
     <add>1.5</add>
+    <html>
+      <type>CheckBox</type>
+      <label>Reserved</label>
+    </html>
   </field>
   <field>
     <name>is_active</name>
-    <title>Option Group Is Active?</title>
+    <title>Option Group Is Active</title>
```

Comment:
FYI I have been waging a bit of a war against the phrase "Is Active" in favor of "Enabled" - I've already changed it in most places but this one seems to have slipped through.